src/screens/Profile/common/types.ts

```diff
@@ -0,0 +1,10 @@
+import { StackNavigationProp } from '@react-navigation/stack'
+import { NavigationScreen } from 'src/common/enums'
+import { RootStackParamList } from 'src/common/types'
+
+type UserScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  NavigationScreen.MY_PROFILE
```

Comment:
```suggestion
  NavigationScreen.PROFILE
```

Reply:
disagree, there will be profiles for users, that is MY PROFILE

Reply:
Ideal name for this - current user profile. But that also could be named as Profile, because it's the one and the only component for this purpose